PyPDF2/_merger.py

```diff
@@ -137,7 +137,7 @@ def merge(
         reader = PdfReader(stream, strict=self.strict)  # type: ignore[arg-type]
         self.inputs.append((stream, reader, my_file))
         if encryption_obj is not None:
-            reader._encryption = encryption_obj
+            reader.encryption = encryption_obj
```

Comment:
Every attribute that does not start with an underscore is part of the public interface. That means we cannot change the behavior / name without deprecation warnings.

Is there a reason why a user would want to access the `encryption` attribute directly?